ios/Podfile.lock

```diff
@@ -1,16 +1,156 @@
+    - GoogleToolboxForMac/NSDictionary+URLArguments (~> 2.1)
+    - GoogleToolboxForMac/NSString+URLArguments (~> 2.1)
+    - GTMOAuth2 (~> 1.0)
+    - GTMSessionFetcher/Core (~> 1.1)
+  - GoogleToolboxForMac/DebugUtils (2.1.3):
+    - GoogleToolboxForMac/Defines (= 2.1.3)
+  - GoogleToolboxForMac/Defines (2.1.3)
+  - GoogleToolboxForMac/NSData+zlib (2.1.3):
+    - GoogleToolboxForMac/Defines (= 2.1.3)
+  - GoogleToolboxForMac/NSDictionary+URLArguments (2.1.3):
+    - GoogleToolboxForMac/DebugUtils (= 2.1.3)
+    - GoogleToolboxForMac/Defines (= 2.1.3)
+    - GoogleToolboxForMac/NSString+URLArguments (= 2.1.3)
+  - GoogleToolboxForMac/NSString+URLArguments (2.1.3)
+  - gRPC (1.10.0):
+    - gRPC-RxLibrary (= 1.10.0)
+    - gRPC/Main (= 1.10.0)
+  - gRPC-Core (1.10.0):
+    - gRPC-Core/Implementation (= 1.10.0)
+    - gRPC-Core/Interface (= 1.10.0)
+  - gRPC-Core/Implementation (1.10.0):
+    - BoringSSL (~> 10.0)
+    - gRPC-Core/Interface (= 1.10.0)
+    - nanopb (~> 0.3)
+  - gRPC-Core/Interface (1.10.0)
+  - gRPC-ProtoRPC (1.10.0):
+    - gRPC (= 1.10.0)
+    - gRPC-RxLibrary (= 1.10.0)
+    - Protobuf (~> 3.0)
+  - gRPC-RxLibrary (1.10.0)
+  - gRPC/Main (1.10.0):
+    - gRPC-Core (= 1.10.0)
+    - gRPC-RxLibrary (= 1.10.0)
+  - GTMOAuth2 (1.1.6):
+    - GTMSessionFetcher (~> 1.1)
+  - GTMSessionFetcher (1.1.14):
+    - GTMSessionFetcher/Full (= 1.1.14)
+  - GTMSessionFetcher/Core (1.1.14)
+  - GTMSessionFetcher/Full (1.1.14):
+    - GTMSessionFetcher/Core (= 1.1.14)
+  - leveldb-library (1.20)
+  - nanopb (0.3.8):
+    - nanopb/decode (= 0.3.8)
+    - nanopb/encode (= 0.3.8)
+  - nanopb/decode (0.3.8)
+  - nanopb/encode (0.3.8)
+  - Protobuf (3.5.0)
 
 DEPENDENCIES:
-  - Flutter (from `/Users/takhion/flutter/bin/cache/artifacts/engine/ios`)
+  - cloud_firestore (from `/Users/niamhpower/.pub-cache/git/flutter_plugins-9cabcc57e63fbc9be4fafee08e6435a1a8c71d61/packages/cloud_firestore/ios`)
+  - firebase_analytics (from `/Users/niamhpower/.pub-cache/hosted/pub.dartlang.org/firebase_analytics-0.3.0/ios`)
+  - firebase_auth (from `/Users/niamhpower/.pub-cache/hosted/pub.dartlang.org/firebase_auth-0.5.1/ios`)
+  - Flutter (from `/Users/niamhpower/flutter/bin/cache/artifacts/engine/ios`)
+  - google_sign_in (from `/Users/niamhpower/.pub-cache/hosted/pub.dartlang.org/google_sign_in-3.0.0/ios`)
 
 EXTERNAL SOURCES:
+  cloud_firestore:
+    :path: /Users/niamhpower/.pub-cache/git/flutter_plugins-9cabcc57e63fbc9be4fafee08e6435a1a8c71d61/packages/cloud_firestore/ios
+  firebase_analytics:
+    :path: /Users/niamhpower/.pub-cache/hosted/pub.dartlang.org/firebase_analytics-0.3.0/ios
+  firebase_auth:
+    :path: /Users/niamhpower/.pub-cache/hosted/pub.dartlang.org/firebase_auth-0.5.1/ios
   Flutter:
-    :path: /Users/takhion/flutter/bin/cache/artifacts/engine/ios
+    :path: /Users/niamhpower/flutter/bin/cache/artifacts/engine/ios
+  google_sign_in:
+    :path: /Users/niamhpower/.pub-cache/hosted/pub.dartlang.org/google_sign_in-3.0.0/ios
```

Comment:
Those paths look sketchy 😄

Reply:
#iostooling

Reply:
Yeah so the podfile.lock should be in version control, but something flutter does here appears to add these paths 🤷‍♀️ https://github.com/flutter/flutter/issues/9299